pkg/operator/ceph/nfs/spec_test.go

```diff
@@ -120,6 +120,9 @@ func TestDeploymentSpec(t *testing.T) {
 						},
 					},
 					PriorityClassName: "my-priority-class",
+					LivenessProbe: &cephv1.ProbeSpec{
+						Disabled: false,
+					},
```

Comment:
Is it possible to also verify that the probe is present in the output?

With adding the ability to override the probe also, I think it would be good to override one of the properties here and check that the output has default values for the probe, except for the overridden value.

Reply:
I added a dedicate test to check livness-probe settings. Naturally, we may check the correctness of `rpcinfo` as liveness-probe only against real nfs-ganesha container.